dist/pom.xml

```diff
@@ -94,6 +108,30 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>net.alchim31.maven</groupId>
+        <artifactId>scala-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>update_config</id>
+            <phase>verify</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <launchers>
+            <launcher>
+              <id>update_rapids_config</id>
+              <mainClass>com.nvidia.spark.rapids.RapidsConf</mainClass>
```

Comment:
Can we have a follow on issue to update RapidsConf so we can find a way to generate the docs with info for multiple versions?  This only works for the current version.

Reply:
https://github.com/NVIDIA/spark-rapids/issues/417